Maddori.Apple/Maddori.Apple/Screen/Home/Home/HomeViewController.swift

```diff
@@ -212,6 +230,72 @@ final class HomeViewController: BaseViewController {
         viewController.modalPresentationStyle = .fullScreen
         present(viewController, animated: true)
     }
+    
+    private func showStartReflectionView() {
+        let viewController = StartReflectionViewController()
+        viewController.modalPresentationStyle = .overFullScreen
+        viewController.dismissChildView = { [weak self] in
+            self?.dismiss(animated: true)
+        }
+        present(viewController, animated: true)
+        // FIXME: - 모달 띄우고 시작하기만 가능한 건 동작을 너무 제한시킴 -> 추가하기 버튼이 채워지면서 시작하기로 바뀌는건 어떨까?
+    }
+    
+    private func convertFetchedKeywordList(of list: [String]) {
+        keywordList = []
+        for i in 0..<list.count {
+            keywordList.append(list[i])
+        }
+        print(keywordList)
+    }
+    
+    // MARK: - api
+    
+    private func fetchCertainTeamDetail(type: HomeEndPoint) {
+        AF.request(type.address,
+                   method: type.method,
+                   headers: type.header
+        ).responseDecodable(of: BaseModel<CertainTeamDetailResponse>.self) { json in
+            if let json = json.value {
+                guard let teamName = json.detail?.teamName else { return }
+                DispatchQueue.main.async {
+                    self.teamNameLabel.setTitleFont(text: teamName)
+                }
+            }
+        }
+    }
+    
+    private func fetchCurrentReflectionDetail(type: HomeEndPoint) {
+        AF.request(type.address,
+                   method: type.method,
+                   headers: type.header
+        ).responseDecodable(of: BaseModel<CurrentReflectionResponse>.self) { json in
+            if let json = json.value {
+                
+                let reflectionDetail = json.detail
+                guard let reflectionDate = reflectionDetail?.reflectionDate?.formatDateString(to: "MM월 dd일 HH시") else { return }
+                guard let reflectionStatus = reflectionDetail?.reflectionStatus else { return }
+                guard let reflectionKeywordList = reflectionDetail?.reflectionKeywords else { return }
```

Comment:
p4;

```suggestion
                guard let reflectionDate = reflectionDetail?.reflectionDate?.formatDateString(to: "MM월 dd일 HH시"),
                let reflectionStatus = reflectionDetail?.reflectionStatus,
                let reflectionKeywordList = reflectionDetail?.reflectionKeywords
                else { return }
```

guard let 을 합쳐서 쓸 수 있습니다 !

Reply:
합쳤습니다![696265d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/116/commits/696265d3b7ffa48d6150fe17feb8efdc5c7c8fef)